src/menubar.ts

```diff
@@ -99,15 +100,22 @@ export default class UI extends EventEmitter {
       updateAvailable: false
     };
 
-    this.incidentFeed.on('update', () => {
-      this.render();
-    });
-
+    // TODO: move this to a dedicated Scheduler
     this.setup().then(() => {
+      // Scheduler should have some method like: doOnce()
+      let first = true;
```

Comment:
I don't feel awesome about adding this flag to the setup function, but I didn't want to pollute the menubar state object. I'm imagining the scheduler can have a "doOnce()" method for handling this sort of thing.